.github/workflows/gh-release.yaml

```diff
@@ -59,9 +59,9 @@ jobs:
         id: check_release_file
         uses: nick-fields/retry@v3
         with:
-          retry_wait_seconds: 10
+          retry_wait_seconds: 20
```

Comment:
This should fix: https://github.com/mondoohq/cnspec/actions/runs/13013349675/job/36296269981